### PR TITLE
Added prototype for ribohmm recipe

### PR DESCRIPTION
**Pull request by eggzilla**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/ribohmm/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+cp -r data/kozak_model.npz $PREFIX/share/kozak_model.npz
+mkdir -p $PREFIX/bin/data/
+cp -r data/kozak_model.npz $PREFIX/bin/data/kozak_model.npz
```
Comment (bgruening) [inline recipes/ribohmm/build.sh]: This line can be removed?

File: recipes/ribohmm/build.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+sed -i 's@data/kozak_model.npz@'"$PREFIX"'/share/kozak_model.npz@' seq.pyx
+$PYTHON setup.py build_ext --inplace --force
```
Comment (bgruening) [inline recipes/ribohmm/build.sh]: Is this really needed?

File: recipes/ribohmm/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+package:
+  name: ribohmm
+  version: "1.0.0"
+
+source:
+  url: https://github.com/rajanil/riboHMM/archive/57a34d5afbd329d9641c895d12d1b0cefea0735b.zip
+  sha256: b0e182e0e07ede199aafef1936e2eab21fa21c6d15ffb1e00386bc0db167b436
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/ribohmm/meta.yaml]: 0

File: recipes/ribohmm/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  skip: True [osx or py3k]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+  host:
+    - python
+    - numpy 1.11
```
Comment (bgruening) [inline recipes/ribohmm/meta.yaml]: I don't think you should pin numpy here.
Is this tool compiling against the numpy ABI?

Comment (eggzilla) [reply recipes/ribohmm/meta.yaml]: Problem is that it failed with newer numpy versions, so i downgraded

**Comment by bgruening** [comment]
The issue is probably upstream. We are trying to fix this upstream.

**Comment by eggzilla** [comment]
Thanks for the help! Could you ping me as soon as it is fixed?

**Pull request closed without merging**